std/traits.d

```diff
@@ -1768,14 +1752,16 @@ Get the function type from a callable object $(D func).
 Using builtin $(D typeof) on a property function yields the types of the
 property value, not of the property function itself.  Still,
 $(D FunctionTypeOf) is able to obtain function types of properties.
---------------------
+
+Example:
+-----------------
 class C
 {
     int value() @property;
 }
 static assert(is( typeof(C.value) == int ));
 static assert(is( FunctionTypeOf!(C.value) == function ));
---------------------
+-----------------
```

Comment:
It didn't link as the value property had no body. I added a dummy body and moved it to a unittest.